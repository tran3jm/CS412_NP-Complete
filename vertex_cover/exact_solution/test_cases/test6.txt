7
A B G
B A C D
C B G
D B F E
F D
E D
G A C