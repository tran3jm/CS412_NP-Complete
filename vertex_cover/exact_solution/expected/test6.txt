Min number: 3
Vertices: 0 2 3